src/aics-image-viewer/components/App/index.tsx

```diff
@@ -719,6 +728,8 @@ export default class App extends React.Component<AppProps, AppState> {
     view3d.setCameraMode(enums.viewMode.VIEW_MODE_ENUM_TO_LABEL_MAP.get(userSelections.mode));
     view3d.setShowAxis(userSelections[SHOW_AXES]);
     view3d.setShowBoundingBox(aimg, userSelections.showBoundingBox);
+    view3d.setBackgroundColor(colorArrayToFloatArray(userSelections.backgroundColor));
```

Comment:
same comment as above, bg color probably should be initialized prior to image load time

Reply:
suggestion: initialize bg color in CellViewerCanvasWrapper/index.jsx, in componentWillReceiveProps (pass bg color in as a prop to this component).   It may also need to be set in componentDidMount also...

Reply:
In this case `setShowAxis` should probably move as well. Looks like everything else that's here and doesn't accept `aimg` as an argument still internally depends on the image having been initialized (looking at `updateExposure`, `setCameraMode`).

Reply:
basically agree, but my thinking was that the axes wont be indicative of much without an image there

Reply:
makes sense, though in an editor configured to display axes by default, having the axes appear with no image at least indicates that the viewer has loaded correctly and any problem is with image loading.